src/account/zendesk.ts

```diff
@@ -40,11 +41,11 @@ export async function sendSupportRequest({
   const uploadTokens = await Promise.all(
     logFiles.map((fileInfo) => _uploadFile(fileInfo, userEmail))
   )
-  const customFields = _generateCustomFields(deviceInfo)
+  const customFields = _generateCustomFields(userProperties)
   await _createRequest({
     message: `${message}
     
-    ${JSON.stringify(deviceInfo)}
+    ${JSON.stringify(userProperties)}
```

Comment:
Wondering if we should indent it, so the support team can more easily read it?
```suggestion
    ${JSON.stringify(userProperties, null, 2)}
```